feat: aggregation mode sdk (#1861)
yetanotherco/aligned_layer: docs/3_guides/1.3_SDK_api_reference_aggregation.md

# Aligned SDK

## API Reference for aggregation mode

### `is_proof_verified_in_aggregation_mode`

> Checks if the proof has been verified with Aligned and is included in the batch on-chain.

Reviewer: ```suggestion
Checks if the proof has been verified with Aligned Aggregation Mode.
```

Author: Solved in [6753e14](https://github.com/yetanotherco/aligned_layer/pull/1861/commits/6753e14fc67850098ffc6d83cdb528b45aee495a).
